src/Admin/SettingsPage/Settings.php

```diff
@@ -112,6 +112,11 @@ public function settings_admin_page_callback()
                             'label'       => __('Disable Impression Tracking', 'mailoptin'),
                             'checkbox_label' => __('Disable', 'mailoptin')
                         ],
+                        'dequeue_google_font'     => [
+                            'type'        => 'checkbox',
+                            'label'       => __('Dequeue Google Fonts', 'mailoptin'),
```

Comment:
@ShoboySnr please change to 
```suggestion
                            'label'       => __('Disable Google Fonts', 'mailoptin'),
```